pkg/api/norman/customization/aks/handler.go

```diff
@@ -209,7 +218,10 @@ func (h *handler) getCredentialsFromBody(req *http.Request, cap *Capabilities) (
 		return http.StatusBadRequest, fmt.Errorf("invalid subscriptionId")
 	}
 	if cap.TenantID == "" {
-		return http.StatusBadRequest, fmt.Errorf("invalid tenantId")
+		cap.TenantID, err = azureutil.FindTenantID(context.Background(), azure.PublicCloud, string(cap.SubscriptionID))
```

Comment:
Going to undo this part, AKSv1 should always have the tenant ID passed in the request body